src/main/java/hudson/plugins/git/GitTool.java

```diff
@@ -137,7 +138,7 @@ public static void onLoaded() {
     }
 
 
-    @Extension
+    @Extension @Symbol("git")
```

Comment:
Who will see the text of that symbol, and how will they see it?

Today, the caller to git-client-plugin defines the desired implementation by passing either `with("git")` or `with("jgit")` or by accepting the default ("git").  If this is visible to users, then "cli-git" is probably clearer than "git" (and better than the word "Default" which is currently displayed as the alternative to "jgit")